### PR TITLE
Normalize command lines in code blocks

### LINKED ISSUE #947: Normalize command lines in code blocks
In most places where there's a command line, it's shown without a prompt, e.g.

```rst
.. code-block:: bash

    python3 --version
```

However, there are some places where it's preceded by a prompt character:

```rst
::

  $ pipx run cowsay moooo
```

I think we should remove the prompt characters entirely, and standardize on `code-block:: bash` (plus `code-block:: bat` for Windows if the command is different). For places where that's not practical (i.e. where including the prompt character and/or command output improves readability), I think we should use `code-block:: console` (plus `code-block:: doscon` for Windows if the command is different).

I think this is a list of all of the occurrences of a prompt character (which notably doesn't include any use of `>` for Windows):

```console
% rg '^\s*(\$ |> )'
source/guides/installing-stand-alone-command-line-tools.rst
47:  $ pipx install PACKAGE
48:  $ ENTRYPOINT_OF_PACKAGE [ARGS]
54:  $ pipx install cowsay
59:  $ cowsay moo
74:  $ pipx list
94:  $ pipx upgrade PACKAGE
95:  $ pipx uninstall PACKAGE
118:  $ pipx run PACKAGE [ARGS]
124:  $ pipx run cowsay moooo
130:  $ pipx --help

source/guides/analyzing-pypi-package-downloads.rst
283:    $ pypinfo requests

source/guides/index-mirrors-and-caches.rst
40:    $ python -m pip wheel --wheel-dir=/tmp/wheelhouse SomeProject
41:    $ python -m pip install --no-index --find-links=/tmp/wheelhouse SomeProject

source/guides/using-testpypi.rst
31:    $ twine upload --repository testpypi dist/*
```

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
- Fixes #947

### CODE REVIEW

File: source/guides/installing-stand-alone-command-line-tools.rst [hunk 1]
```diff
@@ -89,10 +89,10 @@ To see a list of packages installed with pipx and which CLI applications are ava
 
 To upgrade or uninstall the package
 
-::
+.. code::
```
Comment (webknjaz) [inline source/guides/installing-stand-alone-command-line-tools.rst:93]: Why code?

Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: I've noticed that `source/guides/using-testpypi.rst` is using  `.. code::` , so I changed it here. However, I think there is no difference between `::` and `.. code::`, so either one.

Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: @webknjaz If you wish to, I can revoke these changes.

Comment (bhrutledge) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: I don't think `::` or  `.. code::` is accurate. I've pushed a change that implements my suggestion in the #947:

> I think we should remove the prompt characters entirely, and standardize on `code-block:: bash` (plus `code-block:: bat` for Windows if the command is different). For places where that's not practical (i.e. where including the prompt character and/or command output improves readability), I think we should use `code-block:: console` (plus `code-block:: doscon` for Windows if the command is different).

Comment (bhrutledge) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: I'm finding a few more opportunities to use a more semantic `code-block` for prompts. Implementing that now.

**Review by bhrutledge — APPROVED** [review]

File: source/guides/installing-stand-alone-command-line-tools.rst (hunk 1 above)
Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: @bhrutledge So if commands on Unix and Windows are the same, we should always use `.. code-block:: bash` in the future?

Comment (bhrutledge) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: @meowmeowmeowcat That seems to be the precedent.

Comment (meowmeowmeowcat) [reply source/guides/installing-stand-alone-command-line-tools.rst:93]: Thanks for answering!

**Pull request merged**